splice_machine/src/test/java/com/splicemachine/derby/security/SchemaPrivilegeIT.java

```diff
@@ -1275,4 +1281,25 @@ public void testCreateAllPrivilegesToUserTableNotAllowed() throws Exception {
         assertFailed(user2Conn, query, SQLState.AUTH_NOT_OWNER);
 
     }
+
```

Comment:
Could you add some ITs dealing with multiple region servers, similar to those in ClusterDDLTestIT ? If you want to keep them in this class you can tag the test as @Category(HBaseTest) so that it doesn't run in the mem platform

Reply:
Thanks @dgomezferro I'll add the test for multiple region server case. I manually tested with connection to 1527 and 1528, but it is a good idea to put it in IT.

Reply:
I've updated the IT to test two region servers too. Thanks.